packages/auth/src/controllers/teams.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import { validateAll } from 'indicative/validator'
+
+export const createTeam = async (request: Request, response: Response) => {
+    const { manager, body, authUser } = request
+    const { name } = await validate(body)
+
+    const team = await manager('Team').create({
+        name,
+        customer_id: request.authUser!.id,
+    })
+
+    return response.json({ team })
```

Comment:
Please don't return `team` here, because the user can decide to call their own `Squad` or something else. Instead, just return `response.json(team)`